src/content/docs/alerts-applied-intelligence/applied-intelligence/anomaly-detection/anomaly-detection-applied-intelligence.mdx

```diff
@@ -12,33 +12,76 @@ redirects:
 
-For more on data limits, see [Data limits](#data-limits).
+We monitor this unusual behavior in a few ways. First, our team uses the anomaly dashboard so we can see what changed and when. 
+
+<img
+  title="A screenshot of anomaly details in the New Relic UI"
+  alt="A screenshot of anomaly details in the New Relic UI"
+  src={anomalydetails}
+/>
+<figcaption>Explore any anomaly in your system's performance to better understand what errors you're receiving and why.  
+  </figcaption>
+
+We also set up notifications for anomalies to be delivered in Slack, and we set up a webhook to deliver messages when we need them. 
 
-## Why it matters
+<img
+  title="A screenshot of the slack notification set up for anomalies in the New Relic UI"
+  alt="A screenshot of the slack notification set up for anomalies in the New Relic UI"
+  src={anomalyslacknotifications}
+/>
+<figcaption> How to set up slack or webhook notifications for anomalies in the New Relic UI.  
+  </figcaption>
+
+These events are also available for querying, creating custom dashboards, and alerting. After we set up an anomaly detection configuration (a group of apps we're interested in), we can add this configuration as a source. Then the anomalies will be automatically correlated with other data sources via incident intelligence.
+
+## There are two types of anomaly detection: automatic and custom
+
+[Automatic anomalies](/docs/alerts-applied-intelligence/applied-intelligence/anomaly-detection/anomaly-detection-applied-intelligence#automatic-anomalies) are the most efficient way for your team to learn about unusual behavior in your APM-monitored applications. Automatic anomaly detection is a hands-off tool your team can implement to ensure that you're notified the moment behavior in your application deviates from baseline. You can use automatic anomalies to identify the source of the problem and take the appropriate steps to get your system running smoothly again.
 
-With anomaly detection, applied intelligence delivers insights about anomalies in your production system, along with an automatic analysis of the anomaly. It's enabled automatically, at no additional cost. When an anomaly is detected, you can view it in the applied intelligence anomalies feed, or we'll send notifications directly to your Slack channel or a webhook.
+[Custom anomalies](/docs/alerts-applied-intelligence/applied-intelligence/anomaly-detection/anomaly-detection-applied-intelligence#custom-anomalies) allow increased configurability for your team. While custom anomalies require more initial setup than our automatic anomaly option, custom anomalies offer more settings so your team can better control of what signals your team is detecting. Custom anomalies also allow your team the capability to alert on any NQRL condition.
```

Comment:
[Custom anomalies](/docs/alerts-applied-intelligence/applied-intelligence/anomaly-detection/anomaly-detection-applied-intelligence#custom-anomalies) allow increased configurability for your team.  Custom anomalies provide your team with the capability to alert on any NQRL condition and to adjust and optimize your thresholds. Custom anomalies also have all the same advanced tuning settings as static alerting to ensure you are seeing just those anomaly incidents important to you.

Reply:
done!